spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rails_helper'
+
+RSpec.describe 'User logs in', type: :feature do
```

Comment:
Use `let` in specs: https://medium.com/@tomkadwill/all-about-rspec-let-a3b642e08d39